google-cloud-storage/src/main/java/com/google/cloud/storage/StorageClass.java

```diff
@@ -20,8 +20,9 @@
 import com.google.cloud.StringEnumValue;
 
 /**
- * Enums for the storage classes. See https://cloud.google.com/storage/docs/storage-classes for
- * details.
+ * Enums for the storage classes. See <a
+ * href="https://cloud.google.com/storage/docs/storage-classes">https://cloud.google.com/storage/docs/storage-classes</a>
+ * for details.
  */
```

Comment:
I think a better pattern would be to use `@see` annotation and we use only use the actual resource name for link. Something like:

```java
/**
 * Enums for the storage classes.
 *
 * @see <a href="https://cloud.google.com/storage/docs/storage-classes">storage-classes</a> for
 *     more details.
 */
public final class StorageClass extends StringEnumValue {
...
}
```
Applies to this and below changes as well

Reply:
I considered both @see and @link, but came up to direct link.
@see produces "See also" sectin
@link is good to refer to another API member

Now, I think that @see would be better... I will think more.

Thanks for the comment!

Reply:
I replaced direct links with @see for the fields, but not for the class description, which already has 'See also for Serialization form'
the link itself is used for the tag text ( <a href='http://...'>http://...</a> ) intentionally. The generated doc looks better in this manner (imho)

Reply:
I am not sure why we should constrain ourselves from adding multiple `@see` tags. If it looks better in the documentation then also I would suggest you replace the complete URL to resources.  Meaning:

```see <a href="https://cloud.google.com/storage/docs/storage-classes">https://cloud.google.com/storage/docs/storage-classes</a> for details```. 
=> 
```see <a href="https://cloud.google.com/storage/docs/storage-classes">storage classes</a> for details```

Reply:
> I am not sure why we should constrain ourselves from adding multiple @see tags.

We can add multiple see tags, no problem, but this will look not very nice, to me

> I would suggest you at least replace the complete URL to resources

I intentionally use the complete URL. To me it looks more informative
Compare:

Enums for the storage classes. See [storage classes](https://cloud.google.com/storage/docs/storage-classes) for details.

Enums for the storage classes. See https://cloud.google.com/storage/docs/storage-classes for details.

This is a matter of taste :)

Reply:
Yes, that's true 😄